prisma/migrations/20230307210946_init/migration.sql

```diff
@@ -0,0 +1,63 @@
+-- CreateTable
+CREATE TABLE "Example" (
```

Comment:
Why are we migrating this table

Reply:
Just noticed this is what is in the schema, we will be updating this when we update the schema right?

Reply:
Yep, didn't want to remove the example since people might want to inspire from that :)